Makefile

```diff
@@ -21,6 +21,10 @@ prepare-buildx: ## Create buildx builder for multi-arch build, if not exists
 push-extension: prepare-buildx ## Build & Upload extension image to hub. Do not push if tag already exists: make push-extension tag=0.1
 	docker pull $(IMAGE):$(TAG) && echo "Failure: Tag already exists" || docker buildx build --push --builder=$(BUILDER) --platform=linux/amd64,linux/arm64 --build-arg TAG=$(TAG) --tag=$(IMAGE):$(TAG) .
 
+build-helper-images:
+	docker build --platform=linux/amd64 --build-arg copa_version=0.6.2 -t copa-extension container/copa-extension
```

Comment:
can we add copa_version as an argument to the command. For example we would run `make build-helper-images COPA_VERSION=0.6.2`. And the default can be the latest available copa version.

Reply:
added this!